### PR TITLE
Renaming CanTherm as Arkane

### PR DESCRIPTION
**Pull request by alongd**
The name CanTherm no longer represents this software, and we have agreed to rename it as:
`Arkane - Automated Reaction Kinetics and Network Exploration`

This PR refactors CanTherm into Arkane.
Backwards compatibility of input files is conserved, the executable is now `arkane.py` instead of `cantherm.py`.

Examples and documentation were changed accordingly.

(We should also change the labelling of PRs and issues)

### CODE REVIEW

**Comment by alongd** [comment]
I agree that `Arkane` looks better than `ARKANE`, I changed all occurrences (I originally followed the capitalization style of `RMG`, but it doesn't look as good when the word is pronounced with its vowels).

Sure, how do we make it a root level module?

**Comment by mliu49** [comment]
I think you would just move the entire arkane directory up one level to RMG-Py/arkane and update the imports. It might be more trouble than it's worth though, so I think you can decide whether or not you want to try.

**Comment by alongd** [comment]
I'll skip making it a root level module for now, I'll consider it later on.
I think this PR is ready to merge, let me know if you have any further comments

File: documentation/source/conf.py [hunk 1]
```diff
@@ -219,7 +219,7 @@
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [
-  ('latex-rmg', 'RMG-Py_and_CanTherm_Documentation.tex', u'RMG-Py and CanTherm Documentation',
+  ('latex-rmg', 'RMG-Py_and_ARKANE_Documentation.tex', u'RMG-Py and Arkane Documentation',
```
Comment (mliu49) [inline documentation/source/conf.py]: Change capitalization of Arkane here?

File: documentation/source/latex-rmg.rst [hunk 2]
```diff
@@ -11,21 +11,21 @@ steps using a general understanding of how molecules react. This documentation i
 +=============================================+========================================================================================+
 | analyze models & search databases           | `RMG website resources <http://rmg.mit.edu/>`__  (no download needed)                  |
 +---------------------------------------------+----------------------------------------------------------------------------------------+
-| make transition state theory calculations   | Run Cantherm with the Canterm User's Guide                                             |
-+---------------------------------------------+----------------------------------------------------------------------------------------+
 | create mechanisms automatically             | Download RMG with the RMG User's Guide                                                 |
 +---------------------------------------------+----------------------------------------------------------------------------------------+
+| make transition state theory calculations   | Run Arkane after downloading RMG. See Arkane's User's Guide                            |
```
Comment (mliu49) [inline documentation/source/latex-rmg.rst]: Minor, but perhaps `See the Arkane User's Guide` instead to be consistent?

File: examples/arkane/networks/methoxy/input.py [hunk 3]
```diff
@@ -2,11 +2,12 @@
...
 """
-This example illustrates how to manually set up a CanTherm input file for a small P-dep reaction system [using only the
-RRHO assumption, and without tunneling, although this can be easily implemented]. Such a calculation is desireable if the user 
-wishes to supply experimentally determined freqeuncies, for example. Althgou some coommented notes below may be useful, 
-see http://greengroup.github.io/RMG-Py/users/cantherm/index.html for more documented information about CanTherm and 
-creating input files. (information pertaining this file is adopted by Dames and Golden, 2013, JPCA 117 (33) 7686-96.)
+This example illustrates how to manually set up an Arkane input file for a small P-dep reaction system (using only the
+RRHO assumption, and without tunneling, although this can be easily implemented). Such a calculation is desirable if
+the user wishes to supply experimentally determined frequencies, for example. Although some commented notes below may
+be useful, see http://greengroup.github.io/RMG-Py/users/arkane/index.html for more documented information about
```
Comment (mliu49) [inline examples/arkane/networks/methoxy/input.py]: I think this URL should also be updated: http://reactionmechanismgenerator.github.io/RMG-Py/users/cantherm/index.html

File: rmgpy/arkane/main.py [hunk 4]
```diff
@@ -225,7 +222,7 @@ def execute(self):
         
         # Initialize the logging system (both to the console and to a file in the
         # output directory)
-        self.initializeLog(self.verbose, os.path.join(self.outputDirectory, 'cantherm.log'))
+        self.initializeLog(self.verbose, os.path.join(self.outputDirectory, 'Arkane.log'))
```
Comment (mliu49) [inline rmgpy/arkane/main.py]: I would prefer the file name to be completely lowercase, but it's your choice. The gitignore file should also be updated if you decide to change it.

File: rmgpy/arkane/pdepTest.py [hunk 5]
```diff
@@ -43,31 +43,31 @@
 
 
 @attr('functional')
-class CanthermTest(unittest.TestCase):
+class ARKANETest(unittest.TestCase):
```
Comment (mliu49) [inline rmgpy/arkane/pdepTest.py]: Rename class to `ArkaneTest`?

**Comment by mliu49** [comment]
Also, was the HSO2=HSOO example intentionally added in this PR?

File: rmgpy/arkane/main.py (hunk 4 above)
Comment (alongd) [reply rmgpy/arkane/main.py]: Done. Why should `.gitignore` be changed?
Isn't
```python
# Example output files
examples/*
```
enough?

**Comment by alongd** [comment]
Thanks @mliu49 
I addressed all comments, and also made Arkane a root-level module. So far it seems to work smoothly

**Comment by alongd** [comment]
All of Arkane's examples run successfully on this branch

**Comment by alongd** [comment]
@mliu49, this Arkane renaming branch is now rebased and squashed. I believe I addressed all comments, and I also ran all of Arkane's examples on this branch successfully. This PR should be good to go.

**Comment by alongd** [comment]
We should also think about either changing or adding a GitHub label for Arkane

File: examples/arkane/species/C2H4/ethene.log [hunk 6]
```diff
@@ -1826,4 +1826,4 @@
  00000900,0.00001502,0.,0.00005012\\\@
  Job cpu time:  0 days  0 hours  0 minutes 39.5 seconds.
  File lengths (MBytes):  RWF=    329 Int=      0 D2E=      0 Chk=     11 Scr=      1
- Normal termination of Gaussian 03 at Fri Feb  9 00:59:20 2007.
+Normal termination of Gaussian 03 at Fri Feb 9 00:59:20 2007.
```
Comment (mliu49) [inline examples/arkane/species/C2H4/ethene.log]: Were the spacing changes in this file intentional?

Comment (alongd) [reply examples/arkane/species/C2H4/ethene.log]: No. I might have copy-pasted this file, so perhaps the text editor added them. I'll revert it

File: setup.py [hunk 7]
```diff
@@ -208,7 +211,7 @@ def getCanthermExtensionModules():
     for file in files:
         if file.endswith('.py') or file.endswith('.pyx'):
             if 'Test' not in file and '__init__' not in file:
-                if not root.endswith('rmgpy/cantherm/files'):
+                if not root.endswith('rmgpy/arkane/files'):
```
[the remark this answers is not in the document]
Comment (alongd) [reply setup.py]: You're right, something's strange here. The loop in line 208 walks through rmgpy, and seems like setting the module on line 215 should also be changed (it's no longer in rmgpy). I'm not an expert on building/installing python modules, but is it at all needed if arkane is recognized now at the root level?

**Comment by alongd** [comment]
If #1502 is ready, please go ahead with merging it.

**Comment by alongd** [comment]
@mliu49, all modifications were done as we discussed, let me know how the binaries test went

**Comment by mliu49** [comment]
I could compile the binary without any problems and was able to successfully import the arkane module.

Last question, have you compiled the documentation locally to check that it looks fine?

**Comment by alongd** [comment]
Yes, I recompiled and fixed the documentation where it didn't look good

**Comment by alongd** [comment]
Haha :)
Well, @xiaoruiDong already called it Arkane in his talk today... might as well make it so

**Pull request merged**